canonical-web-and-design/maas-ui
# feat(ui): add status and cpu cards to machine details summary overview

## Done

- add status and cpu cards to machine details summary overview

This branch was getting quite large, so `Memory` and `Storage` cards will be in follow-up branches.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Ensure the status and cpu cards render as expected.
- Ensure links in the results list take you to the testing tab.
- With a machine with no tests run, ensure the "Test CPU..." link opens the testing take action button with the appropriate CPU tests selected.

## Fixes

Fixes: # .

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.
